### PR TITLE
update FAQ answer to reference quicksilver hooks.

### LINKED ISSUE #4431: Multidev FAQ does not reflect the availability of Quicksilver hooks
Re: [Multidev FAQ](https://pantheon.io/docs/multidev-faq/)

Priority: Low

## Issue Description
The multidev FAQ doc has the following question and answer:

> Is there a public API available for post-commit hooks or other integrations with external project management systems?
> Not at this time, but it is on the development roadmap.

Quicksilver platform hooks, while not git post-commit hooks, can be used to integrate with external project management systems. It is especially relevant to this doc as there is a Quicksilver hood that runs on multidev creation.

## Suggested Resolution
Revise the post-commit question/answer to surface the availability of the multidev Quicksilver hook.

### PR DESCRIPTION
**Pull request by alexfornuto**
Closes #4431

## Effect
PR includes the following changes:
- Updates Multidev FAQ question about API hooks to link to relevant Quicksilver documentation.

## Remaining Work
- [x] Review from @ataylorme 
- [x] Copy Review

## Post Launch
To be completed by the docs team upon merge: 
- [ ] Update Status Report
- [ ] Archive from **Done** in Waffle

### CODE REVIEW

File: source/_docs/git.md [hunk 1]
```diff
@@ -148,7 +148,7 @@ Now, requests to any `*.drush.in` server address should automatically accept the
 
 ### Checking Out Code using GUI Clients
 
-SourceTree and other Git GUI clients generally prompt for a Source URL using HTTP or HTTPS to the repository to check out the site code. Pantheon does not provide Git repository access over HTTP(s), and instead provides a "Git over SSH" URL. For example:
+BLAG BLAH BLEUSourceTree and other Git GUI clients generally prompt for a Source URL using HTTP or HTTPS to the repository to check out the site code. Pantheon does not provide Git repository access over HTTP(s), and instead provides a "Git over SSH" URL. For example:
```
Comment (ataylorme) [inline source/_docs/git.md]: > BLAG BLAH BLEU

Not sure why this is there

File: source/_docs/terminus/updates.md [hunk 2]
```diff
@@ -28,6 +28,18 @@ mkdir $HOME/terminus
 cd $HOME/terminus
 curl -O https://raw.githubusercontent.com/pantheon-systems/terminus-installer/master/builds/installer.phar && php installer.phar install</code></pre></figure>
   </div>
+
+## Version Life Cycle
```
Comment (ataylorme) [inline source/_docs/terminus/updates.md]: Terminus EOL seems like it should be in its own PR

File: source/_docs/git.md (hunk 1 above)
Comment (alexfornuto) [reply source/_docs/git.md]: Sorry, I was demonstrating how docs changes are staged, and it looks like I hit commit.

File: source/_docs/terminus/updates.md (hunk 2 above)
Comment (alexfornuto) [reply source/_docs/terminus/updates.md]: Yea, looks like I forgot to switch to master first. Resolving...

**Comment by alexfornuto** [comment]
@ataylorme sorry about all the noise, the PR got a little wonky. It's simplified now.
